exports/ystdlib-py/src/ystdlib/find.py

```
@@ -0,0 +1,128 @@
+    1. the path is matched by at least one `include` pattern.
+    2. the path is not matched by any `exclude` patterns.
+    3. the path's filename is matched by at least one `filename` pattern.
+
+    :param include_patterns: pathlib patterns to include paths. `None` or an empty list indicates
+    all paths should be included.
+    :param exclude_patterns: pathlib patterns to exclude paths. `None` or an empty list indicates no
+    paths should be excluded.
+    :param filename_patterns: fnmatch patterns to include filenames. `None` or an empty list
+    indicates all filenames should be included.
+    :return: Matched paths.
+    """
```

Ruff (0.11.9)

First line should end with a period

(D400)